tests/unit/dataactvalidator/test_fsrs.py

```diff
@@ -0,0 +1,56 @@
+from unittest.mock import Mock
+
+from dataactvalidator import fsrs
+
+
+def newClient_call_args(monkeypatch, **config):
+    """Set up and create a newClient request with the provided config. Return
+    the arguments which were sent to the suds client"""
+    mock_client = Mock()
+    monkeypatch.setattr(fsrs, 'Client', mock_client)
+    config = {'fsrs_service': config}
+    monkeypatch.setattr(fsrs, 'CONFIG_BROKER', config)
+    fsrs.newClient()
+    return mock_client.call_args[1]
```

Comment:
Can you talk a bit about what `[1]` is doing here? I looked up the Mock `call_args` function in the docs and it says that it takes a tuple or None. Is this some more pytest magic at work?